pkg/mod/automod.go

```diff
@@ -188,7 +208,11 @@ func updateNotoriety(ctx context.Context, us user.Store, ns NotorietyStore, user
 			if err != nil {
 				return err
 			}
-			notify(ctx, us, user, action)
+			notorietyReport, err := FormatNotorietyReport(ctx, us, ns, user.UUID)
```

Comment:
- Changed to reporting newNotoriety instead. The notification that gets sent despite a SetNotoriety failure did not have a quick fix that I could think of, since we could need to transact across stores. The stores/caches overhaul is becoming more and more urgent (at least to me).
 - Good point, I've changed this.